### ISSUE TITLE
recordselect + rails 4.2: Asset was not declared to be precompiled

### ISSUE THREAD

**Issue by courtland** [activescaffold/active_scaffold#488]
This is specific to the scambra/recordselect gem (latest version).

Similar to issue #477, in rails 4.2 I receive the following error about my asset pipeline configuration:

```ruby
ActionView::Template::Error (Asset was not declared to be precompiled in production.
Add `Rails.application.config.assets.prec
ompile += %w( record_select/next.gif )` to `config/initializers/assets.rb` and restart your server):
```

If I add the config line to my assets.rb it fixes the problem, but it should be configured by active_scaffold.

**Comment by scambra** [comment]
I have fixed it in record_select, 3.4.12

**Comment by courtland** [comment]
Thank you!
